### PR TITLE
WIP Mi scheduler/feature/kebechet merge

### PR DESCRIPTION
**Pull request by xtuchyna**
## Related Issues and Dependencies
Related to https://github.com/thoth-station/mi-scheduler/issues/99

## This introduces a breaking change

- [ ] Yes
- [x] No

## Test or Stage Environment

- [ ] Pull Requests added content to STAGE environment
- [ ] Pull Requests to `AICoE/aicoe-cd` has been opened: 

## This Pull Request implements
New cronjob that schedules kebechet statistics merge

### CODE REVIEW

**Comment by xtuchyna** [comment]
Hello @pacospace could you please have a look at this?

File: mi-scheduler/base/cronjob.yaml [hunk 1]
```diff
@@ -133,8 +131,86 @@ spec:
...
+kind: CronJob
+apiVersion: batch/v1beta1
+metadata:
+  name: mi-scheduler-kebechet-merge
+  labels:
+    app: mi-scheduler-kebechet-merge
+
+spec:
+  schedule: "00 22 * * *"
+  suspend: false
```
Comment (harshad16) [inline mi-scheduler/base/cronjob.yaml:147]: can you suspend all cronjob in base 
and use the cronjob in individual overlays to suspend them.

please also include the cronjob mi-scheduler-kebechet-merge in all the overlays.

**Review by harshad16 — CHANGES_REQUESTED** [review]
all lgtm, just one change.

**Comment by goern** [comment]
@xtuchyna any movement on this? what do we need to do to merge it?

**Comment by xtuchyna** [comment]
#oops this was introduced and fixed in other PR that is already merged https://github.com/thoth-station/thoth-application/pull/1500
We can close this.
/close

**Pull request closed without merging**